xml/dm_registers.xml

```diff
@@ -540,6 +540,50 @@
         <field name="data" bits="31:0" access="R/W" reset="0" />
     </register>
 
+    <register name="Debug Module Control and Status 2" short="dmcs2" address="0x32">
+        This register contains DM control and status bits that didn't easily
+        fit in \Rdmcontrol and \Rdmstatus, and are only likely to be
+        implemented in larger systems.
```

Comment:
what is a "larger" system... you mean one with many harts?

Reply:
Systems that are so large they don't mind an extra hundred gates for debug functionality. Is there a better way to say that?

Reply:
Why not just say "are optional".

Reply:
Done.